final release/model_sklearn.py

```diff
@@ -1,14 +1,37 @@
+"""Creates various Sklearn models"""
+
 import keras 
 import numpy as np
-from sklearn.ensemble import *
-from sklearn.linear_model import *
-from sklearn.svm import *
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.svm import LinearSVC
 
-def model_sklearn(train_x, train_y, method, params):
+def RFT(train_x, train_y, params):
```

Comment:
RFT isn't super clear. `random_forests` would be better. Also, function names should generally start with a lowercase letter.